ProcessMaker/Jobs/ImportScreen.php

```diff
@@ -47,6 +47,14 @@ private function parseFileV2()
         $this->prepareStatus('screens', count($this->file->screens));
         foreach ($this->file->screens as $screen) {
             $new[Screen::class][$screen->id] = $this->saveScreen($screen);
+            //determine if the screen has watchers
+            if (property_exists($screen, "watchers")) {
+                $names = [];
+                foreach ($screen->watchers as $watcher) {
+                    $names[] = $watcher->name;
+                }
+                $this->status['screens']['info'] = __('Please assign a run script user to: ') . implode(', ', $names);
```

Comment:
Could you add the translation in en.json file please? something like:
`"Please assign a run script user to:": "Please assign a run script user to:"`

Reply:
suggestion applied